walkbox.go

```diff
@@ -181,19 +202,34 @@ func (wm *WalkBoxMatrix) nextWalkBox(from, to int) int {
 	return wm.itineraryMatrix[from][to]
 }
 
-// walkBoxAt returns the walk box identifier at the given position or the closest one,
-// along with a boolean indicating inclusion.
-func (wm *WalkBoxMatrix) walkBoxAt(p *Positionf) (id int, included bool) {
-	panic("Not implemented yet!")
+// WalkBoxAt returns the walk box identifier at the given position or the closest one,
+// along with a boolean indicating inclusion. If the point is located between two or more
+// boxes, it returns the lowest walk box ID among them.
+func (wm *WalkBoxMatrix) WalkBoxAt(p *Positionf) (id int, included bool) {
```

Comment:
This function will not be used out from this type. Hence they should be private.